Nodejs/Product/ProjectWizard/ImportWizard.cs

```diff
@@ -29,6 +29,7 @@ public sealed class NewProjectFromExistingWizard : IWizard {
         public void ProjectFinishedGenerating(EnvDTE.Project project) { }
         public void ProjectItemFinishedGenerating(EnvDTE.ProjectItem projectItem) { }
         public void RunFinished() { }
+        public static Boolean AddingNewProject { get; set; } = false;
```

Comment:
* Make this internal
* include this as a property at the top of the file
* no need to set the default value to false because that's the assumed default.